src/components/ServiceDetails/AboutService/DemoToggler.js

```diff
@@ -3,21 +3,21 @@ import { Link } from "react-router-dom";
 import { withStyles } from "@material-ui/styles";
 
 import StyledButton from "../../common/StyledButton";
-import DemoExample from "./DemoExample";
+import ServiceDemo from "./ServiceDemo";
 import Routes from "../../../utility/constants/Routes";
 import { useStyles } from "./styles";
 
 const DemoToggler = ({ classes, showDemo, onClick, service }) => {
   if (showDemo) {
     return (
       <div className={classes.demoContainer}>
-        <h3>Demo Example</h3> <DemoExample service={service} />
+        <h3>Demo Example</h3> <ServiceDemo service={service} />
```

Comment:
Should the heading also change?

Reply:
[Renamed](https://github.com/singnet/refactored-dapp/pull/13/commits/1e5346f84e10e3e1535b53cf9abe477cf6e841fc)